addons/website_sale/controllers/main.py

```diff
@@ -1695,11 +1695,12 @@ def _get_express_shop_payment_values(self, order, **kwargs):
         })
         if request.website.is_public_user():
             payment_form_values['partner_id'] = -1
-        if request.website.enabled_delivery:
-            payment_form_values.update({
+        payment_form_values.update(
```

Comment:
If the `if` is removed, you can directly include the values and skip the `update`.